src/update.c

```diff
@@ -395,8 +395,14 @@ static int main_update()
 	ret = add_included_manifests(server_manifest, &latest_subs);
 	timelist_timer_stop(global_times);
 	if (ret) {
-		ret = EMANIFEST_LOAD;
-		goto clean_exit;
+		if (ret == -add_sub_BADNAME) {
+			/* this means a bundle(s) was removed in a future version */
+			printf("WARNING: One or more installed bundles are no longer available at version %d.\n",
```

Comment:
Seems swupd prints most warnings to stderr. Not asking to change this in this commit, because there are other examples in the code where warnings are printed to stdout.

Reply:
yes, there is an open issue that talks about that #294